client/src/components/Carousel/practice.scss

```
@@ -0,0 +1,43 @@
+@import 'src/styles/package';
+
+.carousel {
+  position: relative;
+  max-width: 450px;
+  @include flex-center();
+  flex-direction: row;
+  overflow: hidden;
+  touch-action: auto;
+
+  .cur-number {
+    position: absolute;
+    bottom: 1em;
+    right: 1em;
+    color: black;
+    background-color: lightgray;
+    padding: 0.3em;
+    border-radius: 0.5em;
+  }
+
+  .left-arrow-btn {
+    position: absolute;
+    left: 1em;
+    color: white;
```

여기도 scss 변수를 사용해주시면 좋을 것 같아요!
```suggestion
    color: $color-white;
```

---

네! 감사해요 ! 사용했습니다~👍